packages/server-islands/playwright.config.ts

```
@@ -0,0 +1,78 @@
+import {
+	defineConfig,
+	devices,
+	type Project,
+	type PlaywrightWorkerOptions,
+} from '@playwright/test';
+
+type Proj = Project<{}, PlaywrightWorkerOptions>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)